### ISSUE TITLE
Tagging a new release anytime soon?

### ISSUE THREAD

**Issue by grembo** [pierresouchay/consul-rust#45]
Hi,

I played a bit with consul-rust. The example for 0.3.0 won't build and changing it so that it builds generates a runtime JSON parse error.

Building and running the services example on master works fine, maybe you could tag 0.4.0, so that something usable becomes available on crates.io?

Thanks

**Comment by red-avtovo** [comment]
Hi @stusmall
do you have any updates on that one?

To all:
meanwhile you can use a library from github directly by providing the dependency like:
```
[dependencies]
consul = { git = "https://github.com/pierresouchay/consul-rust" }
```

**Comment by stusmall** [comment]
@red-avtovo I've handed this project off and I'm not really involved in it anymore.  I was only maintaining it because I was working at a rust/consul heavy shop but have since moved on.  Sorry I can't help

**Comment by pierresouchay** [comment]
@stusmall I tagged the repository, but don't have the right to publish in cargo... would you mind adding me as an owner of the project in Cargo so I can publish a new release?

**Comment by tomers** [comment]
@pierresouchay, I've sent an email to cargo.io, requesting they'll grant you and me permissions for this abandoned crate. Thanks for your help!

**Comment by stusmall** [comment]
@pierresouchay  I'll the cargo access turned over to you.  IIRC the process is pretty quick and straight forward.

**Comment by pierresouchay** [comment]
@stusmall Thank you!

**Comment by pierresouchay** [comment]
@grembo & @tomers Sorry for the delay, I was quite busy those days and thank to @stusmall, I can now publish the crate and work properly.
I just published a new version of the crate (0.4.1)

I'll try to update quickly the CI to work better and I'll try to see at issues
Regards
